.github/workflows/ci.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: "14.x"
+          node-version: "18.x"
```

Comment:
#533 fixed the CI and failing tests. I suggest rebasing on latest `development` to be able to run the new tests in this PR on windows.

Reply:
Done, rebased on top of it! thanks